src/client/editor/Editor.js

```diff
@@ -109,7 +109,8 @@ export default class Editor {
       scene: new THREE.Scene(),
       helperScene: new THREE.Scene(),
       helpers: {},
-      modified: false
+      modified: false,
+      saved: false
```

Comment:
Currently, we determine whether a scene has been saved by checking the existence of `sceneInfo.uri`. 
For example: https://github.com/MozillaReality/Spoke/blob/master/src/client/ui/EditorContainer.js#L296

I would suggest either using that method or using this new flag everywhere.